MetaCell/scicrunch-antibody-registry
# AREG-143 New partner logo

Closes [AREG-143](https://metacell.atlassian.net/browse/AREG-143)

# Implemented solution

Add new partner logo to partners array and add partner's logo image to assets folder
![image](https://github.com/user-attachments/assets/b6ebe3b7-cc9f-4bd6-ab04-ccc2cbd2a7f2)


# How to test this PR
- Run frontend folder of the project
- Go to about page 
- See the partners list

# Sanity checks:

# Breaking changes (select one):

# Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):

### Nice to have (if relevant):


[AREG-143]: https://metacell.atlassian.net/browse/AREG-143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
